### ISSUE TITLE
Feature request: Improving browser addon user experience

### ISSUE THREAD

**Issue by stervel** [omnivore-app/omnivore#2227]
Hello.

This is regarding the Omnivore's Firefox Add-on.  Apologize if this is a duplicate.

Currently I have to use the mouse most of the time when interacting with the browser addon. I think this can be improved.

1. The add-on lets you use Opt/Alt + O, which is great on its own, but I'd also like to be able to access each menu in the dialog (the "edit title - set label - read now") using keyboard too, preferably with a single key (e.g. `e` edit title; `l` labels; `r` read).

2. The Label and Title dialog also don't let me to use Enter button to Save, I have to use the mouse instead just to click the Save button.

3. Title enabling soft return is, I think, unecessary since it's just ... title (refer to no. 2), but this is debatable and varies between use cases. Suggestioin: Shift + Return for linebreaks, just like many chat applications.

4. Label dialog needs to provide suggestions on existing labels so I get a vague reminder of what to type.

**Comment by jacksonh** [comment]
yeah it does follow tabbing convention so you can use tab to change the selected button, and arrows + enter on the labels. I think your idea is still valid though, there's no reason we can't add these keyboard commands, we probably wouldn't use `e` as that is our archive key so we might use it. Probably `i` would work.

**Comment by jacksonh** [comment]
These are all implemented now
